spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +350,20 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public void deleteMultipleItems(String groupId, String citationKeys, Long citationVersion) throws ZoteroConnectionException {
```

Comment:
The method should accept a list of keys and build its own String, not just one string. What if the api changes and the keys are passed differently? It would require changing it everywhere where this method is called instead just this method.